neuralmonkey/attention/feed_forward.py

```diff
@@ -170,6 +166,11 @@ def attention(self,
         return context, next_loop_state
 
     def initial_loop_state(self) -> AttentionLoopState:
+        # TODO blessing
```

Comment:
Co přesně myslíš tím `TODO` jako že tam chceš napsat, proč se to děje?

Reply:
to znamená "udělej s tim něco než se to uzavře". Mělo by tam bejt `TODO(tf-data)` a ne jen TODO.. :-)

Jednak chci dát pryč ten log a dát tam jen debug, nebo ještě něco míň nápadnýho. Plus v komentáři vysvětlit, co to tam dělá a proč to tam musí bejt. Obě dvě věci vyřeším jako response na tenhle review.